src/pudl/metadata/resources/allocate_net_gen.py

```diff
@@ -0,0 +1,196 @@
+                "fuel_consumed_for_electricity_mmbtu",
+            ],
+            "primary_key": [
+                "report_date",
+                "plant_id_eia",
+                "generator_id",
+                "prime_mover_code",
+                "energy_source_code",
+            ],
+        },
+        "field_namespace": "eia",
+        "sources": ["eia923", "eia860"],
+        "etl_group": "outputs",
+    },
+    "generation_fuel_by_generator_monthly_eia923": {
+        "description": (
+            "Monthly estimated net generation by generator. Based on net "
+            "generation reported in the EIA-923 generation and generation_fuel tables."
+        ),
+        "schema": {
+            "fields": [
+                "report_date",
+                "plant_id_eia",
+                "plant_id_pudl",
+                "plant_name_eia",
+                "utility_id_eia",
+                "utility_id_pudl",
+                "utility_name_eia",
+                "generator_id",
+                "unit_id_pudl",
+                "fuel_consumed_for_electricity_mmbtu",
+                "fuel_consumed_mmbtu",
+                "net_generation_mwh",
+            ],
+            "primary_key": [
+                "report_date",
+                "plant_id_eia",
+                "generator_id",
+            ],
+        },
+        "field_namespace": "eia",
+        "sources": ["eia923", "eia860"],
+        "etl_group": "outputs",
+    },
+    "generation_fuel_by_generator_yearly_eia923": {
+        "description": (
+            "Yearly estimated net generation and fuel consumption by generator. Based "
+            "on net generation reported in the EIA-923 generation and generation_fuel "
+            "tables."
+        ),
+        "schema": {
+            "fields": [
+                "report_date",
+                "plant_id_eia",
+                "plant_id_pudl",
+                "plant_name_eia",
+                "utility_id_eia",
+                "utility_id_pudl",
+                "utility_name_eia",
+                "generator_id",
+                "unit_id_pudl",
+                "fuel_consumed_for_electricity_mmbtu",
+                "fuel_consumed_mmbtu",
+                "net_generation_mwh",
+            ],
+            "primary_key": [
+                "report_date",
+                "plant_id_eia",
+                "generator_id",
+            ],
+        },
+        "field_namespace": "eia",
+        "sources": ["eia923", "eia860"],
+        "etl_group": "outputs",
+    },
+    # "generation_fuel_by_generator_energy_source_owner_monthly_eia923": {
+    #     "description": (
+    #         "Monthly estimated net generation and fuel consumption for each generator, "
+    #         "broken down by energy source, prime mover, and owner. Based on data "
+    #         "reported in the EIA-923 generation and generation_fuel tables."
+    #     ),
+    #     "schema": {
+    #         "fields": [
+    #             "report_date",
```

Comment:
should this section go away? why is only the monthly commented out?

Reply:
We're going with just the annual table in the DB until we've got monthly updates to EIA-923 and EIA-860M